ios/FluentUI/Vnext/Button/ButtonTokens.swift

```diff
@@ -265,7 +277,43 @@ open class ButtonTokens: ControlTokens {
         }
     }
 
+    /// Defines the shadow used when a floating button is at rest.
     open var restShadow: ShadowInfo { aliasTokens.elevation[.interactiveElevation1Rest] }
 
+    /// Defines the shadow used when a floating button is pressed.
     open var pressedShadow: ShadowInfo { aliasTokens.elevation[.interactiveElevation1Pressed] }
+
+    /// Defiens the minimum height of the button.
+    open var minHeight: CGFloat {
+        switch style {
+        case .primary, .secondary, .ghost:
+            switch size {
+            case .small:
+                return 28
+            case .medium:
+                return 40
+            case .large:
+                return 52
+            }
+        case .accentFloating, .subtleFloating:
+            switch size {
+            case .small, .medium:
+                return 48
+            case .large:
+                return 56
+            }
+        }
+    }
+
+    /// Defines the minimum vertical padding around the content of the button.
+    open var minVerticalPadding: CGFloat {
```

Comment:
Why is this minimum vertical padding and not just vertical padding?
It seems to me we could have **horizontalPadding** and **verticalPadding** tokens to be more aligned?

Reply:
I think min still makes sense in this case, because the padding doesn't really come into play until the text starts trying to push the height of the button. The default height isn't textsize + verticalPadding*2.

Reply:
FWIW our Figma defines the token as `Min Padding Top/Bottom`, and it kinda makes sense as a minimum value since it may end up larger, depending on the `minHeight` of the button (i.e. there's no guarantee that this will be the exact gap between the text and the outer frame of the button).